### ISSUE TITLE
The v1.12 kubernetes-api docs is missing in the v1.12 branch

### ISSUE THREAD

**Issue by warmchang** [kubernetes/website#10549]
**This is a...** 

- [ ] Feature Request
- [x] Bug Report

**Problem:**
The v1.12 kubernetes-api docs is missing in the v1.12 branch:

https://github.com/kubernetes/website/tree/release-1.12/static/docs/reference/generated/kubernetes-api

![image](https://user-images.githubusercontent.com/13084624/46648098-04d79300-cbc6-11e8-9f7f-26d048275a3a.png)

**Proposed Solution:**
Need to add v1.12 kubernetes-api documentation in v1.12 branch.

**Page to Update:**
https://kubernetes.io/...

/kind bug

**Comment by tengqm** [comment]
The so called 'release-1.12' branch is a snapshot of the working version when k8s 1.12 was not released. When it was released, the v1.12 subdirectory is created and maintained in the 'master' branch. So ... I don't think this is a bug.

**Comment by tengqm** [comment]
Feel free to reopen this if there is a need to create the the 'v1.12' subdir.

**Comment by warmchang** [comment]
@tengqm 
The v1.12 has been released about 11 days ago 
https://github.com/kubernetes/kubernetes/releases/tag/v1.12.0

According to v1.12 Release Schedule, the docs shold be completed by Sep 2018. So I think we should add v1.12 kubernetes-api docs in v1.12 branch.

> tl;dr The 1.12 release cycle begins on Monday, July 2nd, 2018, and ends on release day, Tuesday, September 25, 2018 Thursday, September 27, 2018 . Feature freeze is Tuesday, July 31, 2018. Code freeze begins Tuesday, September 4, 2018 and ends Wednesday, September 19, 2018, but may be vary depending on CI Signal. Docs must be completed and reviewed by Tuesday, September 18, 2018. Release blocking test case additions must be completed by in conjunction with features by code freeze.

Hi @zparnold , what's your suggestion, and have you arranged it (maybe someone is tracking this issue)?
Thanks.
